todo_app/data/trello_items.py

```diff
@@ -0,0 +1,121 @@
+import requests
+import json
+import os
+
+class Cards:
+    def __init__(self, id, name, listName = 'To Do'):
+        self.id = id
+        self.name = name
+        self.listName = listName
+
+    @classmethod
+    def from_trello_card(cls, card, list):
+        return cls(card['id'], card['name'], list)
+
+
+def get_todo():
+    lists = get_lists()
+    # Get the TO DO list id
+    idLists = {}
+    for list in lists:
+        idLists[list['id']] = list['name']
+
+    url = "https://api.trello.com/1/boards/{}/cards".format(os.getenv('TROLLO_BOARD_ID'))
+
+    query = {
+        'key': os.getenv('TRELLO_KEY'),
+        'token': os.getenv('TROLLO_TOKEN'),
+        'cards': 'open'
+    }
+
+    response = requests.request(
+        "GET",
+        url,
+        params=query
+    )
+    response = json.loads(response.text)
```

Comment:
Worth noting that the `response` has a function built in to also do this, but there's no strong preference:
```suggestion
    response = response.json()
```